packages/rocketchat-livechat/.app/client/views/messages.js

```diff
@@ -62,7 +62,7 @@ Template.messages.helpers({
 		};
 	},
 	agentData() {
-		const agent = Livechat.agent;
+		const agent = Template.instance().agentData.get();
```

Comment:
I think this is the same case as `Livechat.guestName` I've talked before (sry for not have seen this before). Any reason to use a `ReactiveVar` here instead of keeping `Livechat. agent` ?